src/sysfiles.c

```diff
@@ -1325,7 +1325,7 @@ void syEchos (
 **  'SyFputs' is called to put the  <line>  to the file identified  by <fid>.
 */
 UInt   syNrchar;                        /* nr of chars already on the line */
-Char   syPrompt [256];                  /* characters already on the line   */
+Char   syPrompt [MAXLENOUTPUTLINE];     /* characters already on the line  */
```

Comment:
So `MAXLENOUTPUTLINE` is 4096 it seems.  Yet in other places, we limit the line length to 32768 (see discussion on #1235) ?

So I am wondering: Does this really resolve those crashes, or just the ones for which we have reproductions? (I.e. did you work through the code logic to verify this eliminates the crash, or does it "just" happen to help in the tested cases?)

Don't get me wrong, I am still fine merging this, I just want to understand how far this really goes :-)